test/data/marriage-abroad-responses-and-expected-results.yml

```diff
@@ -22931,7 +22931,7 @@
   - uk
   - partner_british
   - opposite_sex
-  :next_node: :outcome_opposite_sex_marriage_in_other_countries
+  :next_node: :outcome_opposite_sex_marriage_in_yemen
```

Comment:
Yemen appears in all sets of responses that lead to `outcome_opposite_sex_marriage_in_yemen`, although it's not obvious in the diff because it doesn't show enough context. You can use `git show --unified=5 9fdfc510ae1be39c852097b81a581667ce0d6354 -- test/data/marriage-abroad-responses-and-expected-results.yml` to see that `yemen` is in indeed in the responses.

Does that make sense or have I misunderstood your comment?